lib/features/dashboard/home/comic_list_page.dart

```diff
@@ -0,0 +1,58 @@
+import 'dart:developer';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_modular/flutter_modular.dart';
+import 'package:marvel_app/features/dashboard/response_status.dart';
+import 'package:marvel_app/utils/strings.dart';
+import '../response_status.dart';
+import '../comic_list/comic_list.dart';
+import 'cubit/comic_cubit.dart';
+import 'cubit/comic_list_state.dart';
+
+class ComicListScreen extends StatelessWidget {
+  ComicListScreen({Key? key}) : super(key: key);
+
+  ComicCubit cubit = Modular.get<ComicCubit>();
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocProvider(
+        create: (context) => cubit..getComicList(),
```

Comment:
Huh, I didn't notice it before in your code but if you have cubit or bloc instance injected, it's better to use value instead of create